includes/admin/stripe-settings.php

```diff
@@ -238,6 +250,20 @@
 	}
 	// Insert UPE options below the 'description' setting.
 	$stripe_settings = array_merge( array_splice( $stripe_settings, 0, array_search( 'description', array_keys( $stripe_settings ), true ) + 1 ), $upe_settings, $stripe_settings );
+
+	// in the new settings, "checkout" is going to be enabled by default (if it is a new WCStripe installation).
+	$stripe_settings['payment_request_button_locations']['default'][] = 'checkout';
```

Comment:
I created a migration ticket for this: https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1845